bodhi/client/__init__.py

```diff
@@ -985,6 +985,90 @@ def info_release(name, url, **kwargs):
         print_release(res)
 
 
+@releases.group(name='critpath')
+def critpath_pkgs():
+    # Docs that show in the --help
+    """Interact with critical path packages of a release."""
+    # Developer Docs
+    """Manage the critical path packages of a release."""
+    pass  # pragma: no cover
+
+
+@critpath_pkgs.command(name='list')
+@handle_errors
+@click.argument('name')
+@url_option
+@staging_option
+def list_critpath_packages(name, url, **kwargs):
+    """
+    List critical path packages of a release.
+
+    NAME: Release name (e.g. F27)
+    """
+    client = bindings.BodhiClient(base_url=url, staging=kwargs['staging'])
+
+    res = client.send_request('releases/{}/critpath-packages'.format(name), verb='GET', auth=False)
+
+    if 'errors' in res:
+        print_errors(res)
+    else:
+        print_critpath_pkgs(res)
+
+
+@critpath_pkgs.command(name='add')
+@handle_errors
+@click.argument('name')
+@click.argument('packages', nargs=-1)
+@url_option
+@staging_option
+def add_critpath_packages(name, packages, url, **kwargs):
+    """
+    Add critical path packages to a release.
+
+    NAME: Release name (e.g. F27)
+
+    [PACKAGES]: List of packages to add (e.g. firefox vim python)
+    """
+    client = bindings.BodhiClient(base_url=url, staging=kwargs['staging'])
+
+    packages_str = ','.join(packages)
+    res = client.send_request('releases/{}/critpath-packages'.format(name), verb='PUT',
+                              auth=True, data={"packages": packages_str})
```

Comment:
Let's make this into a binding so that the releng script can use the bindings and not the CLI executable.